src/main/java/duke/UIText.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.util.Scanner;
+
+public class UIText {
```

Comment:
Is there a reason for choosing the name `UIText` instead of `Ui`?

Reply:
I wanted it to only store text outputs for the bot and separate them from the functions which handles the processes.